src/auxinfo/info.rs

```diff
@@ -92,11 +92,12 @@ impl AuxInfoPublic {
     /// modulus N, and that the parameters have appropriate s and t values.
     #[instrument(skip_all, err(Debug))]
     pub(crate) fn verify(&self) -> Result<()> {
```

Comment:
@marsella: For the proofs in Auxinfo that verify that the Pk matches the ZKSetupParameters, is the context supposed to be passed as well ? If so, how is it defined in AuxInfoDecommit::new() ?

Reply:
Yep, those proofs should use the context defined by `AuxInfoParticipant`.

You can update the `Decommit::new` method to either take a `&ProofContext` as parameter or, since multiple of the parameters come from `AuxInfoParticipant`, take `&AuxInfoParticipant` as a parameter and pull out the pid and context internally.